src/vm/inst_dialogue.cpp

```diff
@@ -226,7 +226,6 @@ VmInstruction(InstMesRev) {
   PopUint8(type);
   switch (type) {
     case 0:  // DispInit
-      UI::BacklogMenuPtr->Show();
```

Comment:
Was this already here for MO6 btw?

Reply:
if it was, maybe just add a //TODO or something to change it to update loop based show

Reply:
Yeah, it was already there for MO6, but we also already have update loop-based showing, and that should open the backlog with every `InstMesRev_DispInit` call already (at least in the C;C scripts). I could still ask @MrRevo3D to verify it still works for MO6 though

Reply:
Yeah it, along with *many* other things (unrelated to this PR) have broken. I've reverted this deletion and changed the logic to still work for C;C(LCC) with it, and will get to undoing the rest of the MO6 regression in another PR